.cargo/config.toml

```diff
@@ -3,5 +3,5 @@
 # The following aliases simplify linting the entire workspace
 neon-check = " check  --all --all-targets --features napi-experimental,futures"
 neon-clippy = "clippy --all --all-targets --features napi-experimental,futures -- -A clippy::missing_safety_doc"
-neon-test = "  test   --all               --features=napi-experimental,futures"
-neon-doc = "   rustdoc -p neon            --features=napi-experimental,futures -- --cfg docsrs"
+neon-test = "  test   --all               --features=doc-comment,napi-experimental,futures"
```

Comment:
I don't think `doc-comment` should be here.

Reply:
The reason it's in there is so the doc tests run for the API docs generated by the doc-comment. I verified that this works.